packages/react/src/analytics/integrations/__tests__/__snapshots__/GA4.test.js.snap

```diff
@@ -7,6 +7,7 @@ Array [
     "sign_up_newsletter",
     Object {
       "__blackoutAnalyticsEventId": "111945ad-b9d4-4c21-b3b0-2764b31bd1111",
+      "analytics_package_version": "0.1.0",
```

Comment:
I understand that these are mock values, but shouldn't we ensure these snapshot updates contain both package versions like `core:react`?